src/routes/cluster_workload_info.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python3
+"""Contains the handlers and business logic for the workload API."""
+from collections import defaultdict
+
+import marshmallow
+from connexion.lifecycle import ConnexionResponse
+
+from src.schemas.cluster_workload_info_schema import (
+    ClusterWorkloadInfoSchema,
+    AccountWorkloadResponseSchema,
+    AccountsWorkloadRequestSchema,
+)
+from src.models.cluster_workload_info_model import ClusterWorkloadInfoModel
+
+
+def get_workload_info(body):
+    """Another stub to ensure connexion does not error out."""
+    try:
+        body = AccountsWorkloadRequestSchema().load(body)
+    except marshmallow.exceptions.ValidationError:
+        return ConnexionResponse(status_code=400)
+    workload_data = (
+        ClusterWorkloadInfoModel.query.filter(ClusterWorkloadInfoModel.account_id.in_(body.get("accounts", [])))
+        .order_by(ClusterWorkloadInfoModel.account_id)
```

Comment:
That would result in an alphabetical order of uuid fields and then most of the time one account would only have one cluster associated with it.

But since your reasoning is correct as well, I think I should go with the equivalent of `ORDER BY  account_id, cluster_id` and order by both as that seems better from a response perspective?

Reply:
I've made the change this way, let me know if it's fine now